modules/volume/rendering/renderabletimevaryingvolume.cpp

```diff
@@ -317,7 +332,8 @@ void RenderableTimeVaryingVolume::loadTimestepMetadata(const std::string& path)
     try {
         ghoul::Dictionary dictionary = ghoul::lua::loadDictionaryFromFile(path);
         metadata = RawVolumeMetadata::createFromDictionary(dictionary);
-    } catch (...) {
+    }
+    catch (...) {
```

Comment:
It's really dangerous to just catch everything here ... it's better to figure out which exceptions you actually expect (heh) to get and then only catch these